src/CG-DNA/bond_oxdna2_fene.cpp

```diff
@@ -24,8 +25,8 @@ using namespace LAMMPS_NS;
 void BondOxdna2Fene::compute_interaction_sites(double e1[3], double e2[3], double /*e3*/[3],
                                                double r[3]) const
 {
-  constexpr double d_cs_x = -0.34;
```

Comment:
Any idea what removing the `constexpr` keyword does? I presume setting these through setter functions doesn't allow it.